src/test/librbd/mirror/snapshot/test_mock_CreatePrimaryRequest.cc

```diff
@@ -167,10 +167,10 @@ class TestMockMirrorSnapshotCreatePrimaryRequest : public TestMockFixture {
   void expect_unlink_peer(MockTestImageCtx &mock_image_ctx,
                           MockUnlinkPeerRequest &mock_unlink_peer_request,
                           uint64_t snap_id, const std::string &peer_uuid,
-                          int r) {
+                          bool expect_peer_removal, int r) {
```

Comment:
Nit: I'd call it `is_linked`, as the peer itself isn't actually removed in the mock.